contracts/JBFundingCycleStore.sol

```diff
@@ -465,9 +505,19 @@ contract JBFundingCycleStore is JBControllerUtility, IJBFundingCycleStore {
     // Get the necessary properties for the base funding cycle.
     JBFundingCycle memory _baseFundingCycle = _getStructFor(_projectId, _fundingCycle.basedOn);
 
+    // Find the base cycle that is not still queued.
+    while (_baseFundingCycle.start > block.timestamp) {
+      // Set the configuration.
+      configuration = _baseFundingCycle.configuration;
+      // Get the funding cycle for the configuration.
+      _fundingCycle = _getStructFor(_projectId, configuration);
```

Comment:
`configuration` and `_fundingCycle` aren't needed in this loop, we can get these after we find the correct `_baseFundingCycle`.

Reply:
Isn't this the same as `_fundingCycle = _baseFundingCycle;`?